lib/player.rb

```diff
@@ -0,0 +1,9 @@
+class Player
+  attr_reader :name 
+  attr_accessor :choice
+
+  def initialize(name)
```

Comment:
When you initialize two global variables here, do you need two parameters? How does the player register their choice? (a genuine question, not saying your code is wrong!)

Reply:
Agreed - where doe the choice come from? If the choice is not known at initialisation, then you could do `@choice = nil` instead, to show it's nil by default.